File: app/src/main/java/com/github/nickpakhomov/hotel_viewer/ui/activities/DetailPresenter.java
package com.github.nickpakhomov.hotel_viewer.ui.activities;

import android.database.Cursor;
import android.support.v4.app.LoaderManager;

import com.google.android.gms.maps.OnMapReadyCallback;

/**
 * Created by Nikolay Pakhomov on 08/09/17.
 */

interface DetailPresenter extends
        LoaderManager.LoaderCallbacks<Cursor>,
        OnMapReadyCallback,
        Presenter {
    void onDestroy();
}

File: app/src/main/java/com/github/nickpakhomov/hotel_viewer/ui/activities/DetailPresenter.kt
package com.github.nickpakhomov.hotel_viewer.ui.activities

import android.database.Cursor
import android.support.v4.app.LoaderManager

import com.google.android.gms.maps.OnMapReadyCallback

/**
 * Created by Nikolay Pakhomov on 08/09/17.
 */

internal interface DetailPresenter : LoaderManager.LoaderCallbacks<Cursor>, OnMapReadyCallback, Presenter {
    fun onDestroy()
}
